src/component/HeroCard.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { IHero } from '../config/type/hero';
+
+const useStyle = makeStyles(() => ({
+  card: {
+    height: '100%',
+  },
+  cardMedia: {},
+  cardContent: {
+    flexGrow: 1,
+  },
+}));
+
+const dumm: IHero = {
```

Comment:
Do not hard code data, try to use https://www.npmjs.com/package/axios

Reply:
Yes, I add hard code to see how data is displayed at front end. Fix it now!